guides/get-going-with-gitops.mdx

```diff
@@ -0,0 +1,485 @@
+description: Configuring Flipt for a GitOps workflow
+---
+
+## Why GitOps?
+
+GitOps is a set of practices centered around storing your application and infrastructure configuration in Git.
+The goal being to leverage the capabilities of version-control for your entire systems configuration.
+
+If you're already embracing GitOps practices for your other configuration, then aligning your feature flags with these same practices can complete the experience.
+No more correlating what changed in Git, with what changed in your feature flag system to understand the entire state of the world.
+Git is the single source of truth as GitOps intended.
+
+So how do we achieve this with Flipt?
+
+## What We Will Learn
+
+In this guide we will:
+
+- 🏁 Add a feature flag to an existing codebase
+- 📝 Define the flag via Flipt's configuration format
+- 🌲 Add, commit and push the change to a production serving branch
+- 🎯 Adjust our configuration to target an internal group of users in our organization
+- 🌓 Progressively enable the flag for proportions of the user base
+
+## Setting the Scene
+
+Our guide starts with an imaginary organization, with a single web application defined in Go and committed to a single GitHub repository.
+We as the developer, have been tasked to experiment with a new sorting algorithm on an endpoint for our application.
+
+Our application handles requests from authenticated users.
+These users also happen to be grouped into organizations.
+We will use this information in our targeting rules later on.
+
+This endpoint happens to list out a bunch of strings in a JSON array.
+The sorting algorithm previously used was slow (Bubble Sort), and we want to try something new (Quicksort).
+While we feel confident in our implementation, we're going to practice caution and release the change behind a feature flag.
+
+### The Application
+
+```bash
+.
+├── go.mod
+├── go.sum
+├── main.go
+└── pkg
+    └── server
+        └── words.go
+```
+
+Our application lives in a directory committed to a Git repository.
+For the examples sake, we assume the repository will be hosted on GitHub at https://github.com/organization/repository.git.
+
+The target of our change is a `http.HandlerFunc` definition (in the file `pkg/server/words.go`) with the name `ListWords`.
+Currently, the function uses a sorting function `bubblesort` and we're going to swap this for the `quicksort` function.
+
+```go pkg/server/words.go
+func (s *Server) ListWords(w http.ResponseWriter, r *http.Request) {
+  words, err := getWords(r.Context())
+  if err != nil {
+    http.Error(w, err.Error(), http.StatusInternalServerError)
+    return
+  }
+
+  bubblesort(words)
+
+  if err := json.NewEncoder(w).Encode(words); err != nil {
+    http.Error(w, err.Error(), http.StatusInternalServerError)
+    return
+  }
+}
+```
+
+### Calling Flipt
+
+Instead of calling `bubblesort` directly, we're going to use the Flipt Go SDK to switch this call based on the feature flag `use-quicksort-algorithm`.
```

Comment:
```suggestion
Instead of calling `bubblesort` directly, we're going to use the [Flipt Go SDK](https://github.com/flipt-io/flipt/blob/main/sdk/go) to switch this call based on the feature flag `use-quicksort-algorithm`.
```